src/components/Thread/__tests__/Thread.test.js

```diff
@@ -84,7 +84,7 @@ describe('Thread', () => {
       <Chat client={chatClient}>
         <TranslationProvider value={{ ...translators, t }}>
           <Channel channel={channel} client={chatClient} thread={thread2}>
-            <Thread thread={thread2} />
+            <Thread />
```

Comment:
Can we update thread in context between these renders? I think the point of the test here is to test the thread changing.

Reply:
yeah so passing thread values into `Channel` is what changes the context value of thread that's being used in Thread